x/stake/types/delegation.go

```diff
@@ -24,6 +24,30 @@ type delegationValue struct {
 	Height int64
 }
 
+// defines a delegation without type Rat for shares
+type DelegationWithoutRat struct {
+	DelegatorAddr sdk.AccAddress `json:"delegator_addr"`
+	ValidatorAddr sdk.AccAddress `json:"validator_addr"`
+	Shares        string         `json:"shares"`
+	Height        int64          `json:"height"`
+}
+
+// aggregates of all delegations, unbondings and redelegations
+type DelegationSummary struct {
+	Delegations          []DelegationWithoutRat `json:"delegations"`
+	UnbondingDelegations []UnbondingDelegation  `json:"unbonding_delegations"`
+	Redelegations        []Redelegation         `json:"redelegations"`
+}
+
+func NewDelegationWithoutRat(delegation Delegation) DelegationWithoutRat {
```

Comment:
Is this just for returning in JSONs?  Doesn't the sdk.Decimal MarshalJSON to string already?

Reply:
yeah pretty much. The voyager team had an issue with that #2096.

Reply:
Idk if it's still a problem, but I'd prefer to wait for the fee dist to be implemented to do the refactor and tests